### PR TITLE
Refactor createTopic

### PR DESCRIPTION
**Pull request by shvaikalesh**

### CODE REVIEW

File: topics.js [hunk 1]
```diff
@@ -1,19 +1,16 @@
...
-    const $button = document.createElement('button');
     $li.className = "topic-tag-action f6 float-left js-tag-input-tag";
+    const $button = document.createElement('button');
     $button.className = "delete-topic-button f5 no-underline ml-2 js-remove";
-    $button.setAttribute('tabIndex', '-1');
-    $button.innerHTML = '×';
-    $input = document.createElement("input");
+    $button.tabIndex = -1;
+    $button.textContent = "×";
+    const $input = document.createElement("input");
```
Comment (shvaikalesh) [inline topics.js:9]: Globals leak. I suggest `use strict`.

File: topics.js [hunk 2]
```diff
@@ -1,19 +1,16 @@
...
     const $li = document.createElement('li');
-    const $text = document.createTextNode(keyword);
-    const $button = document.createElement('button');
     $li.className = "topic-tag-action f6 float-left js-tag-input-tag";
+    const $button = document.createElement('button');
     $button.className = "delete-topic-button f5 no-underline ml-2 js-remove";
-    $button.setAttribute('tabIndex', '-1');
-    $button.innerHTML = '×';
-    $input = document.createElement("input");
+    $button.tabIndex = -1;
```
Comment (shvaikalesh) [inline topics.js:7]: Setters ftw.

File: topics.js [hunk 3]
```diff
@@ -1,19 +1,16 @@
...
+    $button.textContent = "×";
+    const $input = document.createElement("input");
     $input.name = "repo_topics[]"
     $input.value = keyword;
     $input.hidden = true;
-    $li.appendChild($text);
-    $li.appendChild($button);
-    $li.appendChild($input);
+    $li.append(keyword, $button, $input);
```
Comment (shvaikalesh) [inline topics.js:13]: `append` wraps strings in text nodes.

File: topics.js [hunk 4]
```diff
@@ -1,19 +1,16 @@
 (function () {
   const createTopic = (keyword) => {
     const $li = document.createElement('li');
-    const $text = document.createTextNode(keyword);
```
Comment (shvaikalesh) [inline topics.js:4]: There is also `new Text(keyword)` btw

File: topics.js (hunk 2 above)
Comment (yavorsky) [reply topics.js:7]: yeah, props to attrs reflection 💡 thanks

**Review by yavorsky — APPROVED** [review]

**Pull request merged**